App/containers/AllEventsScreen/index.tsx

```diff
@@ -18,11 +19,32 @@ import api from '../../utils/api';
 import {NavigationStackProp} from 'react-navigation-stack';
 import dayjs from 'dayjs';
 import {ApiResponse} from 'apisauce';
+import { AssignEventPayload } from 'App/typings';
 
 type Props = {
   navigation: NavigationStackProp;
 };
 const AllEvents = ({navigation}: Props) => {
+  const assignToEvent = (data: object) => () => {
+    const { id } = data;
+        api.assignEvent({ eventId:id }).then(({ok, data}: ApiResponse<any>) => {
+          if (ok) {
+            Toast.show({
+              type: 'success',
+              text: 'Przypisałeś się do wydarzenia',
+              buttonText: 'Ok', 
+            });
+            navigation.navigate('PrivateStack');
+          } else {
+            Toast.show({
+              type: 'danger',
+              //@ts-ignore
+              text: data.message || 'Już się przypisałeś do wydarzenia',
```

Comment:
mamy pewnosc ze to bedzie blad?  powinnismy wyswietlic blad z przypisaniem

Reply:
zmieniłam, weź jak możesz to looknij :D